docs/csharp/language-reference/keywords/checked-and-unchecked.md

```diff
@@ -13,24 +13,22 @@ ms.assetid: a84bc877-2c7f-4396-8735-1ce97c42f35e
 # Checked and Unchecked (C# Reference)
 C# statements can execute in either checked or unchecked context. In a checked context, arithmetic overflow raises an exception. In an unchecked context, arithmetic overflow is ignored and the result is truncated.  
   
--   [checked](../../../csharp/language-reference/keywords/checked.md) Specify checked context.  
+-   [checked](checked.md) Specify checked context.  
   
--   [unchecked](../../../csharp/language-reference/keywords/unchecked.md) Specify unchecked context.  
-  
- If neither `checked` nor `unchecked` is specified, the default context depends on external factors such as compiler options.  
+-   [unchecked](unchecked.md) Specify unchecked context.  
   
  The following operations are affected by the overflow checking:  
   
 -   Expressions using the following predefined operators on integral types:  
   
-     `++` `--` - (unary)   `+` -   `*` `/`  
+     `++`, `--`, unary `-`, `+`, `-`, `*`, `/`  
   
 -   Explicit numeric conversions between integral types.  
   
- The [/checked](../../../csharp/language-reference/compiler-options/checked-compiler-option.md) compiler option lets you specify checked or unchecked context for all integer arithmetic statements that are not explicitly in the scope of a `checked` or `unchecked` keyword.  
+ If neither `checked` nor `unchecked` is specified, the default context is defined by the value of the [-checked](../compiler-options/checked-compiler-option.md) compiler option. That option lets you specify checked or unchecked context for all integer arithmetic statements that are not explicitly in the scope of a `checked` or `unchecked` keyword.  
```

Comment:
It's useful to add here that if the -checked compiler option is not explicitly specified, arithmetic operations occur in an unchecked context.

Reply:
@rpetrusha that's a good addition. Should I also clarify the default behavior for constant expressions (quote from the specification):
>For constant expressions (expressions that can be fully evaluated at compile-time), the default overflow checking context is always `checked`.

Reply:
Yes, @pkulikov, that's also a really useful addition (and I had no idea)!